content/project/contents+ar.lr

```diff
@@ -0,0 +1,19 @@
+_model: page
+---
+_template: beeware.html
+---
+title: مشروع BeeWare
+---
+summary: ما هو مشروع BeeWare ؟
+---
+sort_key: 3
+---
+gutter:
+
+لنرحّب ببروتوس
+===========
+.. image:: /static/images/brutus-128.png
+
+بروتوس النحلة هو شعار مشروع BeeWare . هو نحلة عاملة مشغولة.
+
+Fun fact: a person who keeps bees is called an **apiarist** - a name that the BeeWare project uses to describe the core team. This also reflects the relationship that the core team wishes to have with the community - we're shepherds of the community, and we'll provide direction, but we know the real work is done by the community as a whole.
```

Comment:
Lets get this fun fact going and I think we should be good to go :-), thanks again for all the hard work!

Reply:
Done, and I'm sorry I couldn't help more than that for the moment.